Registry/Registry.EM_COMMON_add

```diff
@@ -0,0 +1,3367 @@
+# At the present time this file is managed manually and edited by hand.                                         
```

Comment:
@BarryHLynn 
Barry,
What is the purpose of this file? Is it specific to lightning?